docs/manual/macros.md

```diff
@@ -49,14 +49,14 @@ At the end of invocation of a parameterized macro, the above macros are
 
 Within the body of a macro, there are several constructs that permit
 testing for the presence of optional parameters. The simplest construct
-is "%{-f}" which expands (literally) to "-f" if -f was mentioned when the
-macro was invoked. There are also provisions for including text if flag
-was present using "%{-f:X}". This macro expands to (the expansion of) X
-if the flag was present. The negative form, "%{!-f:Y}", expanding to (the
+is `%{-f}` which expands (literally) to "-f" if -f was mentioned when the
```

Comment:
Shouldn't the two latter -f's be back-quoted as well?

Reply:
(Fixed the third `-f`.)

Reply:
Fair enough, the line what should be back-quoted is indeed blurry.